plugins/source-kafka/README.md

```diff
@@ -53,26 +53,26 @@ Manage your Knative Kafka eventing sources
 
 #### SEE ALSO
 
-* [kafka create](#kafka-create)	 - create NAME
-* [kafka delete](#kafka-delete)	 - delete NAME
-* [kafka describe](#kafka-describe)	 - describe NAME
+* [kafka create](#kafka-create)	 - Create a kafka source
+* [kafka delete](#kafka-delete)	 - Delete a Kafka source
+* [kafka describe](#kafka-describe)	 - Describe a Kafka source
 
 ### kafka create
 
-create NAME
+Create a kafka source
 
 #### Synopsis
 
-create NAME
+Create a kafka source
 
 ```
-kafka create NAME [flags]
+kafka create NAME --sink SINK [flags]
```

Comment:
I think all the flags except sink are required for kafka source creation.

Reply:
also, usage should also include being with `kn source`, i.e. 
```
kn source kafka create NAME --servers SERVERS --topics TOPICS --consumergroup GROUP [options]
```

Reply:
I have added all flags to the usage output.
As to add `kn source` to the usage, it's not easy to fix here. This part of README.md is auto generated by `cobra/doc`.  I could try to change my doc generator tool to add `kn source` in front of command usage. Let's keep it as it in this PR. I will create a new issue.